main.go

```diff
@@ -20,9 +20,10 @@ import (
 )
 
 const (
-	dcoCheck = "dco_check"
-	comments = "comments"
-	deleted  = "deleted"
+	dcoCheck  = "dco_check"
+	comments  = "comments"
+	deleted   = "deleted"
```

Comment:
Maybe the `opened` const could go here if it's not present already?

Reply:
added the const in the `pullRequestHandler.go` file as that's the only place it's used. Similar to these consts which are only used in the `main.go` file.

Happy to move it if you'd prefer all the const definitions are in one place.